collection-scripts/gather_etcd_pprof

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+#Safeguards
+set -o nounset
+set -o errexit
+set -o pipefail
+
+MIN_TIME="$(date -u -d '24 hours ago' +%Y-%m-%dT%H:%M:%SZ)"
+
+BASE_COLLECTION_PATH="/must-gather"
+ETCD_LOG_PATH="${BASE_COLLECTION_PATH}/etcd_info"
+
+mkdir -p "${ETCD_LOG_PATH}"
+
+CEO_POD=$(oc get pods -n openshift-etcd-operator -l app=etcd-operator --no-headers | awk '{print $1}')
+
+if [ -z "${CEO_POD}" ]; then
+    echo "ERROR: No running etcd-operator pods found" >&2
+    exit 1
+fi
+
+echo "INFO: Gathering CEO pprof goroutine dump from pod ${CEO_POD}"
+
+oc exec -n openshift-etcd-operator "${CEO_POD}" -- curl -s http://127.0.0.1:6060/debug/pprof/goroutine?debug=2 > "${ETCD_LOG_PATH}/ceo_pprof_goroutine_dump.txt"
```

Comment:
I think, instead of having a separate collection script, we can add these changes in gather_etcd_more?. That would be few lines of code in there and users can get pprofs for free when they run `gather_etcd_more`.